sonar-db/src/main/java/org/sonar/db/issue/IssueChangeDao.java

```diff
@@ -36,6 +37,7 @@
 import static java.util.Arrays.asList;
 
 @ServerSide
+@ComputeEngineSide
 public class IssueChangeDao implements Dao {
```

Comment:
No need for `@ServerSide` nor `@ComputeEngineSide` on DB components (DAOs, DefaultPeriodCleaner, PermissionRepository). As annotations are not systematically used, I suggest to remove all of them.